assets/src/block-editor/components/with-media-library-notice.js

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
This is copied almost verbatim from [with-cropped-featured-image.js](https://github.com/ampproject/amp-wp/blob/bcda6d639920c8fe5c5e5fa5af83ef15d815a72f/assets/src/common/components/with-cropped-featured-image.js#L1).